cluster-up/common.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+#
+# This file is part of the Kepler project
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Copyright 2022 The Kepler Contributors
+#
+
+set -e
+
+# the cluster kind is a kubernetes cluster
+if [ ${CLUSTER_PROVIDER} = "kind" ]; then
```

Comment:
I don't want `CLUSTER_PROVIDER` check been appears in many shell scripts. as duplicate validation?

Reply:
I understand the concern, we could set this validation directly in the other scripts.
I just wanted to make it a generic function, for example, we will need later to also say that `microshit` is an `openshift` cluster.
But I agree that we can do it in a different way...